backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/serialization/KonanDeclarationTable.kt

```diff
@@ -15,7 +15,3 @@ class KonanDeclarationTable(builtIns: IrBuiltIns, descriptorTable: DescriptorTab
         loadKnownBuiltins()
     }
 }
-
-// This is what we pre-populate tables with.
-val IrBuiltIns.knownBuiltins
```

Comment:
@alexander-gorshenev As I understood this is already on common part of serialization in Kotlin.

Reply:
yep.